TestResultSummaryService/routes/getPerffarmRunCSV.js

```diff
@@ -5,14 +5,16 @@ const path = require('path');
 const ArgParser = require('../ArgParser');
 module.exports = async ( req, res ) => {
     const {user, pass} = ArgParser.getConfig() === undefined ? {} : ArgParser.getConfig().perfFarm;
-    request.get('http://perffarmServer/perfsite/services/parse_results.php?metric=&runids='+req.query.baselineID+'&ignoreExcludes=true&format=csv&groupby=run', {
+    const {server} = ArgParser.getConfig() === undefined ? {} : ArgParser.getConfig().perfFarmServer;
```

Comment:
Could we have data structured like the following in the config file?

```
	"perfFarm": {
		"server": "server url";
		"user" : "username",
		"pass" : "password"
	},
```

In this way, we just need to use one line to get it:
const {server, user, pass} = ArgParser.getConfig() === undefined ? {} : ArgParser.getConfig().perfFarm;